### DISCUSSION TITLE
survey-likert Question Prompt Font Size

### DISCUSSION THREAD

**Discussion by GABowers** [jspsych/jsPsych#2863] [General]
I want to change the size of the font across my entire experiment. This works with `.jspsych-display-element { font-size: 35px; }` for most elements but does not for the survey-likert plugin. I added a couple of options I found that changed the text for the options on the scale, but not the prompt itself.

My code:

>
          <style>
            .jspsych-display-element {
              font-size: 35px;
            }
            .jspsych-survey-likert-statement {
                font-size: 50px;
            }
            .jspsych-survey-likert-opt-label {
                font-size: 35px;
            }
          </style>

Again - prompt text font size for the questions does not change. I inspected the element in my browser (Edge) and saw the following.

![likert_font](https://user-images.githubusercontent.com/15266221/201686024-7257f2ec-04d2-4a55-a12d-721867ae0a5a.png)

My modified styles seem to be overridden by the default. I'm using JSPsych 7.3.0 and all of my plugins are 1.1.1.

What do I need to do to fix this?

**Comment by ChristopheBossens** [comment]
You can try adding the !important syntax after specifying the font-size, like so:

```
.jspsych-survey-likert-statement {
            font-size: 50px !important;
}
```

**Reply by GABowers** [reply]
This solved the issue. Thanks!
